source/_integrations/launch_library.markdown

```diff
@@ -7,30 +7,48 @@ ha_iot_class: Cloud Polling
 ha_release: 0.83
 ha_codeowners:
   - '@ludeeus'
+  - '@DurgNomis-drol'
 ha_domain: launch_library
 ha_platforms:
   - sensor
 ---
 
 The `launch_library` sensor will provide you with information about the next planned space launch.
 
-## Configuration
+{% include integrations/config_flow.md %}
 
-Add the data to your `configuration.yaml` file as shown in the example:
+The data this platform presents comes from [launchlibrary.net][launchlibrary].
+
+## Using in Automations
+
+Example below will send a notification 15 minutes before liftoff.
+
+{% raw %}
 
 ```yaml
-# Example configuration.yaml entry
-sensor:
-  - platform: launch_library
+alias: Notify me 15 minutes before liftoff
+description: 'Send me a notification 15 minutes before liftoff.'
+trigger:
+  - platform: template
+    value_template: >-
+      {{ (as_timestamp(states('sensor.launch_time')) - 900) <=
+      as_timestamp(now())  }}
+condition: []
+action:
+  - service: notify.mobile_app_ac2003
+    data:
+      title: Rocket launch in less the 15 minutes!
+      data:
+        ttl: 0
+        priority: high
+      message: >-
+        {{ states('sensor.next_launch') }} - at <b>{{
+        as_timestamp(states('sensor.launch_time')) |
+        timestamp_custom('%H:%M', false) }}</b>, launch probability {{ states('sensor.launch_probability') }}%.
+mode: single
```

Comment:
Let make it a little simpler and more readable:)

```suggestion
- alias: "Notify me 15 minutes before liftoff"
  trigger:
    - platform: template
      value_template: >-
        {{ (as_datetime(states("sensor.launch_time") - now()).seconds <= 900) }}
action:
  - service: notify.mobile
    data:
      title: "Rocket launch in less the 15 minutes!"
      message: >-
        {{ states("sensor.next_launch") }}
        - at {{ as_datetime(states("sensor.launch_time")).time() }},
        launch probability {{ states("sensor.launch_probability") }}%.
```

Reply:
I really don't think this should be in here.
If you want to share this automation it should be a blueprint here instead <https://community.home-assistant.io/c/blueprints-exchange/53>

Reply:
Totally okay, the only reason I did it, is because other integration often provide an example for users and thought this one was a good example on how user could use this integration in a smart way 😊

Reply:
I have now removed it 😃